include/secutils/certstatus/certstatus.h

```diff
@@ -132,4 +132,12 @@ bool check_cert_revocation(X509_STORE_CTX* ctx, OPTIONAL OCSP_RESPONSE* resp);
  */
 int check_revocation_any_method(X509_STORE_CTX* ctx);
 
+/*
+ * Check revocation status on certs in ctx->chain. As a generalization of
+ * check_revocation() in crypto/x509/x509_vfy.c, considers local CRLs only.
```

Comment:
Note that `check_revocation()` in `crypto/x509/x509_vfy.c` does only consider local CRLs.
So as far as this function is needed at all (see further commens below), just say:
```
* Check revocation status on certs in ctx->chain considering local CRLs only.
```